ckanext/publications_qld_theme/plugin.py

```diff
@@ -48,10 +59,10 @@ def is_request_for_resource():
 # this ensures external css/js is loaded from external staging if running in cicd/pdev environments.
 def set_external_resources():
     environment = config.get('ckan.site_url', '')
-    if 'ckan' in environment:
-        return '//staging.data.qld.gov.au'
-    else:
+    if 'data.qld' in environment:
         return ''
+    else:
```

Comment:
should this have publications instead of data?

Reply:
It's piggybacking on the CMS files for Data, since Publications doesn't have its own CMS setup.